src/components/App/index.js

```diff
@@ -81,7 +82,10 @@ class App extends React.Component {
     /* TODO: we shouldn't put all the params in the state: some of them aren't
      * needed because are stored in models, and other need to be parsed */
     /* Here we update general state with router params and our device check. */
-    const newParams = Object.assign({}, { donation: this.router.params.attributes.donation && true }, this.router.params.attributes);
+    const newParams = Object.assign({}, {
```

Comment:
The order in the `Object.assign` method is wrong, what you need is:
```javascript
const newParams = Object.assign({},
  this.router.params.attributes,
  {
    donation: !!this.router.params.attributes.donation,
    embed: !!this.router.params.attributes.embed
  });
```

In the future, we should also consider to not include all of the router params but instead pick the one we're interested in (and parse them if necessary), as written in the TODO statements above.